src/main/resources/swagger/api-docs.yaml

```diff
@@ -11017,3 +11021,13 @@ components:
             email: email authorization
             profile: profile authorization
             https://www.googleapis.com/auth/cloud-billing: GCS billing
+    oidc:
```

Comment:
Second oauth flow based on authorizationCode. The current `implicit` flow is still available.